meta-mender-sunxi/recipes-bsp/u-boot/files/0003-Enable-env-is-in-mmc-define-offset.patch

```diff
@@ -0,0 +1,32 @@
+From 889810f7a0d0766d18440cb615ac962a8a21ac59 Mon Sep 17 00:00:00 2001
+From: Marek Belisko <marek.belisko@open-nandra.com>
+Date: Mon, 10 Dec 2018 18:07:26 +0100
+Subject: [PATCH] Enable env is in mmc + define offset
+
+Signed-off-by: Marek Belisko <marek.belisko@open-nandra.com>
+---
+ configs/orangepi_zero_defconfig | 2 ++
+ 1 file changed, 2 insertions(+)
+
+diff --git a/configs/orangepi_zero_defconfig b/configs/orangepi_zero_defconfig
+index 0da9c65..2b585c9 100644
+--- a/configs/orangepi_zero_defconfig
++++ b/configs/orangepi_zero_defconfig
+@@ -1,5 +1,6 @@
+ CONFIG_ARM=y
+ CONFIG_ARCH_SUNXI=y
++CONFIG_ENV_OFFSET=0x800000
```

Comment:
This is due to the fact that "config ENV_OFFSET" was left visible. But this actually problematic as meta-mender is supposed to set ENV_OFFSET based which partition alignment you chose.

Reply:
Yes I should properly review patches ;). Thanks.